lib/dash/dash_parser.js

```diff
@@ -271,8 +271,18 @@ shaka.dash.DashParser = class {
   async requestManifest_() {
     const requestType = shaka.net.NetworkingEngine.RequestType.MANIFEST;
     const type = shaka.net.NetworkingEngine.AdvancedRequestType.MPD;
+    const getManifestUris = () => {
```

Comment:
Is there any reason for this to be a function that is called immediately?

Reply:
To simplify use